src/components/Experiment/QuickOutput/QuickOutput.js

```diff
@@ -231,6 +233,15 @@ export default function QuickOutput(givenProps) {
               />
             </>
           );
+        case audioToAudio:
+          return (
+            <>
```

Comment:
Do you need the extra `<>` around `<AudioToAudioOutput>`?

Reply:
My bad, I was testing something out bcs of which I added the fragment

Reply:
Removed